README.md

```diff
@@ -15,10 +15,28 @@
 `test`: contains the test source code files and where test binaries are moved to.
 `tools`: contains tools and scripts, some used by tool.py
 
+## Dependencies
+
+- [CMake](https://cmake.org/) (3.13+)
```

Comment:
It's probably worth noting that the python tool can install cmake for you as well